docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair`
+
+Demographic parity can be used to assess the extent of allocation harms, as it 
+reflects an assumption that resources should be allocated proportionally 
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on assumptions: that either the dataset is not a good representation of 
+what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the world, but the world is unjust. 
+
+It's worth considering whether the assumptions underlying this measurement 
+model maintain construct validity (see :ref:`construct_validity`). 
+Consequential validity, for example, asks how the world shaped by using the 
+measurement model; in this case, the relevant question would be whether 
+demographic parity meets the criteria for establishing "fairness", itself 
+an unobservable theoretical construct, and whether satisfying demographic 
+parity brings us closer to the world we'd like to see. 
+
+Equalized odds
+~~~~~~~~~~~~~~
+A classifier :math:`h` satisfies equalized odds under a
+distribution over :math:`(X, A, Y)` if its prediction :math:`h(X)` is
+conditionally independent of the sensitive feature :math:`A` given the label
+:math:`Y`. This is equivalent to
+:math:`\E[h(X) \given A=a, Y=y] = \E[h(X) \given Y=y] \quad \forall a, y`.
+:footcite:`agarwal2018reductions` Equalized odds requires that the true 
+positive rate, :math:`\P(h(X)=1 | Y=1`, and the false positive rate, 
+:math:`\P(h(X)=1 | Y=0`, be equal across groups. 
+
+The inclusion of false positive rates acknowledges that different groups 
+experience different costs from misclassification. For example, in the case of 
+a model predicting a negative outcome (e.g., probability of recidivating) 
+that already disproportionately affects members of minority communities, 
+false positive predictions reflect preexisting disparities in outcomes 
+across minority and majority groups. Equalized odds further enforces that the 
+accuracy is equally high across all groups, punishing models that only 
+perform well on majority groups.
+
+Equalized odds and can be used to diagnose both allocation harms as well as 
+quality-of-service harms. It can be useful for diagnosing allocation harms 
```

Comment:
I would given an example of allocation harm (e.g., the typical example of hiring) and quality of service harm (e.g., gender shades). For allocation harms, I think it is good to highlight how this metric differs from demographic parity (i.e., equalized odds considers not the allocation of resources, but, for lack of a better term, the allocation of "deserved" resources, where "deserved" is determined by the target variable). 

I think this will make the point of the assumption that the target variable is a good measurement of the phenomena being modeled clearer as well.

Reply:
Added the following details with new text in bold:

> Equalized odds and can be used to diagnose both allocation harms as well as quality-of-service harms. **Allocation harms occur when AI systems allocate opportunities, resources, or information differently across different groups (for example, an AI hiring system that is more likely to advance resumes of male applicants than resumes of female applicants regardless of qualification). Quality-of-service harms occur when an AI system does not work as well for one group versus another (for example, facial recognition systems that are more likely to fail for dark-skinned individuals). For more information on AI harms, see 
:ref:`types_of_harms`.** 

Equalized odds can be useful for diagnosing allocation harms 
because its goal is to ensure that a machine learning model works equally well for different groups. **Another way to think about equalized odds is to contrast it with demographic parity. While demographic parity assesses the allocation of resources generally, equalized odds focuses on the allocation of resources that were actually distributed to members of that group (indicated by the positive target variable Y=1).** However, equalized odds makes the assumption that the target variable :math:`Y` is a good measurement of the phenomena being modeled, but that assumption may not hold if the measurement does not satisfy the requirements of construct validity.